RELEASE_PROCEDURE.md

```diff
@@ -11,31 +11,29 @@ When you release fixed version of SpotBugs, please follow these procedures.
 ## Release to Maven Central
 
 When we push a tag, the build result on GitHub Actions will be deployed to the [SonaType Nexus](https://oss.sonatype.org/), and published to the Maven Central automatically by [Gradle Nexus Publish Plugin](https://github.com/gradle-nexus/publish-plugin). Then we can find [artifacts in the Maven Central](https://repo1.maven.org/maven2/com/github/spotbugs/) after several hours.
+See the `build` phase in `.github/workflows/release.yml` calling the `publishToSonatype` and `closeSonatypeStagingRepository` gradle tasks.
 
 ## Release to Eclipse Update Site
 
-It's automated by Github CI.
+It's automated by GitHub CI.
 
 When we push tag, the build result will be deployed to [eclipse-candidate repository](https://github.com/spotbugs/eclipse-candidate).
 When we push tag and its name doesn't contain `_RC`, the build result will be deployed to [eclipse repository](https://github.com/spotbugs/eclipse).
 
-See `deploy` phase in `.github/workflows/release.yml` for detail.
+See `Deploy eclipse`, `Deploy eclipse-candidate` and `Deploy eclipse-latest` phases in `.github/workflows/release.yml` for detail.
 
 ## Release to Eclipse Marketplace
 
 Update version in [Eclipse Marketplace page](https://marketplace.eclipse.org/content/spotbugs-eclipse-plugin). If you have no permission, please contact with @KengoTODA or @iloveeclipse.
 
 ## Release to Gradle Plugin Portal
```

Comment:
I think this part is done by the [spotbugs-gradle-plugin](https://github.com/spotbugs/spotbugs-gradle-plugin), but I'm not sure. Can someone please verify it?

Reply:
yes that is the gradle plugin, it releases far more frequently.

Reply:
I mean this project, spotbugs/spotbugs does not release anything to the gradle plugin portal, neither manually or automatically, right? I found [com.github.spotbugs at the gradle plugin portal](https://plugins.gradle.org/plugin/com.github.spotbugs), but this is the one released by the [spotbugs-gradle-plugin](https://github.com/spotbugs/spotbugs-gradle-plugin).
However, this project does release to the maven portal, but the maven plugin itself is released by the [spotbugs-maven-plugin](https://github.com/spotbugs/spotbugs-maven-plugin).
If this is true, I will modify the release procedure file to reflect this.